library/src/main/java/com/github/pwittchen/prefser/library/PreferencesAccessorsProvider.java

```diff
@@ -48,7 +48,7 @@ private void createAccessors() {
   private void createBooleanAccessor() {
     accessors.put(Boolean.class, new Accessor<Boolean>() {
       @Override public Boolean get(String key, Boolean defaultValue) {
-        return preferences.getBoolean(key, defaultValue);
+        return preferences.getBoolean(key, (defaultValue == null) ? false : defaultValue);
```

Comment:
In this use case, we are defining default behavior for the user in the case of passing `null`. We don't know if a user wants `true` or `false`, but we're forcing using one of these. The same rule applies to all of the changes below.